### DISCUSSION TITLE
Screener module examples?

### DISCUSSION THREAD

**Discussion by unprobable-x** [ranaroussi/yfinance#2114] [Q&A] [answered]
Any example queries for how to make use of the [Screener module](https://github.com/ranaroussi/yfinance?tab=readme-ov-file#market-screener)?

**Comment by ValueRaider** [comment, marked as answer]
New documentation on the way, sneak peek here: https://ericpien.github.io/yfinance/reference/yfinance.sector_industry.html

**Reply by unprobable-x** [reply]
Amazing! TY!

**Reply by johncast14** [reply]
It looks amazing! Thank you for the documentation!
